despatch_advice_import_ubl/wizard/despatch_advice_import.py

```diff
@@ -125,6 +129,25 @@ def parse_ubl_despatch_advice_line(self, line, ns):
             "uom": {"unece_code": qty_xpath[0].attrib.get("unitCode")},
             "backorder_qty": backorder_qty,
         }
+
+        package_id_xpath = line.xpath(
+            "cac:Shipment/cac:TransportHandlingUnit/cac:ActualPackage/cbc:ID",
+            namespaces=ns,
+        )
```

Comment:
these are elements, not xpaths 
```suggestion
        package_id_el = line.xpath(
            "cac:Shipment/cac:TransportHandlingUnit/cac:ActualPackage/cbc:ID",
            namespaces=ns,
        )
```

Reply:
Hi @simahawk , I added 1 commit to replace that